### ISSUE TITLE
Unable to replace text in a textframe

### ISSUE THREAD

**Issue by femo1de** [ShapeCrawler/ShapeCrawler#332]
1. Prepare a PPT and create a new autoshape with two string (inside the same shape) that you can identify as placeholders
2. With ShapeCrawler (current version), load the presentation, load that autoshape, retrieve the textbox 
3. Replace both your placeholders via TextFrame.Text = "my new value"
4. Check that the textframe contains the changed text

### Expected
- The Textframe contains both changed strings
or
- An exception is thrown

### Actual
No exception is thrown and the textframe still contains the old text

I have added a PR with a UnitTest that demonstrates this #333

**Comment by femo1de** [comment]
@adamshakhabov Do you know what causes this or a workaround? I can help with that but right now I dont know whats causing it

**Comment by femo1de** [comment]
seems this caused by some sort of caching / memory handling inside shaprecrawler.

I'm using a template like this: 
![image](https://user-images.githubusercontent.com/90273092/193412404-df5589b0-d82d-4f38-9ec9-b12122ace232.png)

Closing and reopening the presentation between operations works and will replace both placeholders in the shape:

```
using var pres = SCPresentation.Open(targetPath);
{
	var autoShape = pres.Slides[2].Shapes.GetByName<IAutoShape>("Text Placeholder 4");
	autoShape.TextFrame.Text = autoShape.TextFrame.Text.Replace("{{customerneed}}", "replace with customer need");
	pres.Save();
	pres.Close();
}
using var pres2 = SCPresentation.Open(targetPath);
{
	var autoShape = pres2.Slides[2].Shapes.GetByName<IAutoShape>("Text Placeholder 4");
	autoShape.TextFrame.Text = autoShape.TextFrame.Text.Replace("{{se_managernote}}", "replace with manager note");
	pres2.Save();
	pres2.Close();
}
```

However this code will replace only the second placeholder:

```
using var pres = SCPresentation.Open(targetPath);
{
	var autoShape = pres.Slides[2].Shapes.GetByName<IAutoShape>("Text Placeholder 4");
	autoShape.TextFrame.Text = autoShape.TextFrame.Text.Replace("{{customerneed}}", "replace with customer need");
	pres.Save(); 
// add breakpoint here
	autoShape.TextFrame.Text = autoShape.TextFrame.Text.Replace("{{se_managernote}}", "replace with manager note");
	pres.Save();
	pres.Close();
}
```

Using a breakpoint in the second example, you can actually see that the presentation is first saved with the first placeholder replaced and then overwritten with only the second one replaced. 

It seems that something inside SC caches the TextFrame.Text contents and those do not get updated when the text is replaced. Maybe something to do with the use of Lazy?

**Comment by femo1de** [comment]
Found the issue and have added a fix to PR #333
